src/main/java/net/mcreator/ui/component/JStringListField.java

```diff
@@ -121,51 +91,34 @@ public JStringListField(Window parent, @Nullable Function<VTextField, Validator>
 		edit.setContentAreaFilled(false);
 		edit.setToolTipText(L10N.t("components.string_list.edit"));
 		edit.addActionListener(e -> {
-			List<String> newTextList = ListEditorDialog.open(parent, textList, validator, uniqueEntries);
+			List<String> newTextList = ListEditorDialog.open(parent, getTextList(), validator, uniqueEntries);

```

Comment:
Should I rebuild this to accept `Enumeration<String>` instead?

Reply:
May make sense yes

Reply:
✔️ Fixed